plio/tests.py

```diff
@@ -135,3 +135,106 @@ def setUp(self):
     def test_for_question(self):
         # write API calls here
         self.assertTrue(True)
+
+
+class ImageTestCase(PlioTestCase):
+    """Tests the Image CRUD functionality."""
+
+    def setUp(self):
+        # seed some plios
+        super().setUp()
+
+        # seed some items and questions
+        self.plio_1 = Plio.objects.filter(name="Plio 1").first()
+
+        self.item_1_plio_1 = Item.objects.create(
+            plio=self.plio_1, type="question", time=1
+        )
+        self.item_2_plio_1 = Item.objects.create(
+            plio=self.plio_1, type="question", time=2
+        )
+
+        self.question_1_plio_1 = Question.objects.create(item=self.item_1_plio_1)
+        self.question_2_plio_1 = Question.objects.create(item=self.item_2_plio_1)
+
+    def test_user_can_attach_images_to_their_question(self):
+        """
+        Tests whether a user can link images to the questions that were created by themselves
+        """
+        # upload a test image and retrieve the id
+        with open("plio/static/plio/test_image.jpeg", "rb") as img:
+            response = self.client.post(reverse("images-list"), {"url": img})
+        uploaded_image_id = response.json()["id"]
+
+        # update the question with the newly uploaded image
+        response = self.client.put(
+            reverse("questions-detail", args=[self.question_1_plio_1.id]),
+            {"item": self.item_1_plio_1.id, "image": uploaded_image_id},
```

Comment:
item does not need to passed in put, no? it only updates whatever is being passed - so, item should remain unchanged if not passed.

Reply:
<img width="537" alt="Screenshot 2021-06-15 at 8 53 41 PM" src="https://user-images.githubusercontent.com/20169829/122080254-c6d1a900-ce1b-11eb-8861-52812a0c7ef8.png">